docs/src/examples.md

```diff
@@ -86,6 +86,63 @@ julia> round.(vcov(ols); digits=5)
   0.38889  -0.16667
  -0.16667   0.08333
 ```
+By default, the `lm` method uses the Cholesky factorization which is known as fast but numerically unstable, especially for ill-conditioned design matrices. You can use the `method` keyword argument to apply a more stable QR factorization method.
```

Comment:
I think it would be more accurate to also state that with Cholesky factorization near-collinear columns are dropped (as this is the most common issue, the case where Cholesky is not accurate enough also can happen but is much rarer).

Reply:
I have updated the example. Now there are two examples, aggressive multicollinearity detection and unstable estimates.